### DISCUSSION TITLE
[Impovement] Enable requesting team as an assignee on Issues

### DISCUSSION THREAD

**Discussion by bobbyphtr** [community/community#86679] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

It would be great if we could assign teams with a round-robin (your choice) system as an assignee in an issue. Hence we can have an equal load on handling issues.

Ps.
There are lots of aspirations to enable this feature. Looks like its been neglected for almost 7 years+...
Would be great if GitHub could take a look at this!

https://github.com/dear-github/dear-github/issues/170

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by philm001** [comment]
I would like +1 up vote this improvement

**Comment by hughesjordan** [comment]
This would be incredibly useful and its really needed in order to help maintain a design system.

**Comment by RCheesley** [comment]
+1 this is a really important aspect of being able to triage and assign issues efficiently, especially in larger projects!

**Comment by rajbos** [comment]
Plus one on this. There is so few attention for adding teams to anything, which is why it’s not used anywhere. Assigning PRs and issues to a team makes sure they all will see it (notifications) and one of them can pick it up. Especially when you’re a contributor to a lot of repos, it’s currently very easy to miss new work being requested across repos. I currently have an [action](https://github.com/devops-actions/issue-comment-tag) specifically for this that just adds a comment to tag the team 🙃. Otherwise people just have to happen to stumble upon a new issue/PR. (This is also why we already tag people directly, only adding more work)

**Comment by molson504x** [comment]
I'm also going to +1 on this.  I've had a few times now, especially with using IssueOps, where being able to assign a PR or an Issue to a team would greatly improve the process.  I don't necessarily want to assign a team as a reviewer on a PR, but I'd rather assign the team to the PR as an assignee for visibility purposes.  Same goes with issues - when an issue comes in, it should be able to be assigned to a team for triage and handling rather than having to assign it to a project (which is also not necessarily team-based) or an individual(s).

**Comment by philm001** [comment]
@jmeridth 

Can we as a group have an update on this? As can be seen from this repo:

https://github.com/dear-github/dear-github/issues/170

And even the comments here, this is a hotly requested feature and it would benefit small teams (free) and large teams (enterprise) enormously. This feature has been requested for about 7 years

**Reply by samus-aran** [reply]
Hi there @philm001 👋 ,

Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) for more information.

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.

Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.

**Reply by philm001** [reply]
Thank you for the message. I apologize for any offense. Thank you for posting a link to the code of conduct. I have reviewed it again and I am now aware of this.

I do hope though that GitHub will take the feedback here in this thread and others into implementing the requested feature and that this can be under serious consideration.

**Comment by nunodsfernandes** [comment]
Hi team,

Any updates on this?

7+ year old thread: https://github.com/dear-github/dear-github/issues/170
No feedback.

Near 1 year old thread here, and the only feedback we get is someone telling us that we should not bother GH staff...
Is there any other way to get you guys to listen and provide feedback?

If we had this kind of dev support on our company, for sure the team would have already been "rotated out".

**Listen to your community!**
